### ISSUE TITLE
Add AGPL-3.0 license

### ISSUE THREAD

**Issue by luislucena16** [nostr-protocol/nostr#182]
hey nostr devs, is there a reason why you don't have an [OSS licence](https://www.fsf.org/licensing/) added? it seems ideal to add it :)
I'm looking forward to do it if you approve it!

**Comment by luislucena16** [comment]
yes, but I think it could still be added, also to the other repos

**Comment by BitcoinBrief** [comment]
> There is no code for Nostr itself, it is just an idea and it's on the public domain.

Ideas need protection from patent trolls and other rent seekers. May be worthwhile exploring some copyleft options to keep the ideas in the public domain.
